core/domain/platform_parameter_domain_test.py

```diff
@@ -1865,7 +2138,16 @@ def test_serialize_and_deserialize_returns_unchanged_platform_parameter(
             'rules': [
                 {
                     'filters': [
-                        {'type': 'server_mode', 'conditions': [['=', 'prod']]}],
+                        {
+                            'type': 'server_mode',
+                            # The expected type of 'conditions' key is
+                            # List[Tuple[str, str]], but for testing purposes
+                            # we are providing value of type List[List[str]].
+                            # Which causes MyPy to throw error, Thus to silent
+                            # the error we used ignore here.
+                            'conditions': [['=', 'prod']]  # type: ignore[list-item]
```

Comment:
In this test, the dictionary is first converted to JSON string, and then again converted to python object. So, while converting it from a python object to a JSON string, `Tuple` is converted to a normal List because in JSON I think there is no concept of Tuples. So, if we change this list to a tuple and the backend test fails because the expected output from the JSON string is LIST.